man/build.q.set.Rd

```diff
@@ -11,31 +11,34 @@
 
 \arguments{
   \item{q.concourse}{
-  	A matrix with meaningful short handles as rownames, languages as columns, and full items in cells as produced by \code{\link{import.q.concourse}}.
+  	A matrix with (meaningful, short) handles as rownames, languages (if applicable) as columns, and full items in cells as produced by \code{\link{import.q.concourse}}.
   }
   \item{q.sample}{
-  	A character vector of sampled meaningful short handles, such as \code{c("q-is-great","q-is-obscure")}.
+  	A character vector of sampled (meaningful, short) handles, such as \code{c("q-is-great","q-is-obscure")}.
   }
   \item{q.distribution}{
-  	The chosen q distribution as a vector of integers, such as \code{c(1,3,1)}.
+  	The chosen Q distribution as a vector of integers, such as \code{c(1,3,1)}.
   }
 }
 \details{
-	Q studies are carried out letting participants rank a \emph{sample} of statements (items), collectively referred to as the \emph{q-set}.
-	These q-sets are drawn (by some sampling strategy) from  \emph{concourse}, or universe of items.
+	Q studies are carried out letting participants rank a \emph{sample} of statements (items), collectively referred to as the \emph{Q set}.
+	These Q sets are drawn (by some sampling strategy) from a \emph{concourse}, or universe of items.
 	This function currently simply subsets the concourse generated by \code{\link{import.q.concourse}} and returns it as \code{q.set}.
```

Comment:
This function subsets the concourse generated by \code{\link{import.q.concourse}}, based on a vector of handles provided, and returns it as \code{q.set}.